lib/modules/dosomething/dosomething_gladiator/dosomething_gladiator.module

```diff
@@ -56,19 +46,41 @@ function _dosomething_gladiator_build_http_client() {
 function dosomething_gladiator_send_user_to_gladiator($user, $values) {
   $client = _dosomething_gladiator_build_http_client();
 
```

Comment:
yeah, so the way it was constructed before was relying on guzzle to load, and creating an object, now it's just returning an array, no need!